platform/iphone/ios.mm

```diff
@@ -30,39 +30,90 @@
 
 #include "ios.h"
 
+#import <StoreKit/StoreKit.h>
 #import <UIKit/UIKit.h>
 
 void iOS::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("get_rate_url", "app_id"), &iOS::get_rate_url);
+	ClassDB::bind_method(D_METHOD("show_store_rating_ui"), &iOS::show_store_rating_ui);
 };
 
 void iOS::alert(const char *p_alert, const char *p_title) {
-	UIAlertView *alert = [[[UIAlertView alloc] initWithTitle:[NSString stringWithUTF8String:p_title] message:[NSString stringWithUTF8String:p_alert] delegate:nil cancelButtonTitle:@"OK" otherButtonTitles:nil, nil] autorelease];
-	[alert show];
+	NSString *titleString = [NSString stringWithUTF8String:p_title];
+	NSString *messageString = [NSString stringWithUTF8String:p_alert];
+
+	// Create an alert controller with an OK button for these strings
+	UIAlertController *controller = [UIAlertController alertControllerWithTitle:titleString
+																		message:messageString
+																 preferredStyle:UIAlertControllerStyleAlert];
+	UIAlertAction *okAction = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault handler:nil];
+	[controller addAction:okAction];
+
+	// Finally present the alert
+	[[UIApplication sharedApplication].keyWindow.rootViewController presentViewController:controller
+																				 animated:YES
+																			   completion:nil];
 }
 
 String iOS::get_rate_url(int p_app_id) const {
-	String templ = "itms-apps://ax.itunes.apple.com/WebObjects/MZStore.woa/wa/viewContentsUserReviews?type=Purple+Software&id=APP_ID";
-	String templ_iOS7 = "itms-apps://itunes.apple.com/app/idAPP_ID";
-	String templ_iOS8 = "itms-apps://itunes.apple.com/WebObjects/MZStore.woa/wa/viewContentsUserReviews?id=APP_ID&onlyLatestVersion=true&pageNumber=0&sortOrdering=1&type=Purple+Software";
-
-	//ios7 before
-	String ret = templ;
-
-	if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 7.0 && [[[UIDevice currentDevice] systemVersion] floatValue] < 7.1) {
-		// iOS 7 needs a different templateReviewURL @see https://github.com/arashpayan/appirater/issues/131
-		ret = templ_iOS7;
-	} else if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 8.0) {
-		// iOS 8 needs a different templateReviewURL also @see https://github.com/arashpayan/appirater/issues/182
-		ret = templ_iOS8;
-	}
-
-	// ios7 for everything?
-	ret = templ_iOS7.replace("APP_ID", String::num(p_app_id));
-
-	printf("returning rate url %ls\n", ret.c_str());
-	return ret;
+	String rate_url = "itms-apps://itunes.apple.com/WebObjects/MZStore.woa/wa/viewContentsUserReviews?id=APP_ID&onlyLatestVersion=true&pageNumber=0&sortOrdering=1&type=Purple+Software";
+	rate_url = rate_url.replace("APP_ID", String::num(p_app_id));
+	printf("returning rate url %ls\n", rate_url.c_str());
+	return rate_url;
+};
+
+void iOS::show_store_rating_ui() {
+	[SKStoreReviewController requestReview];
```

Comment:
Apple's [latest solution](https://developer.apple.com/documentation/storekit/skstorereviewcontroller?language=objc) to asking for reviews.